src/main/resources/messages.properties

```
@@ -47,6 +47,10 @@ empty.password=Password field is required
 empty.confirm.password=Confirm password field is required
 empty.email=Email field is required
 
+blank.topic=Topic is blank
+blank.description=Description is blank
+blank.wishes.to.exchange=Wishes to exchange is blank
```

It would be much better to have more informative messages:
```suggestion
blank.topic=Topic field should not be blank
blank.description=Description should not be blank
blank.wishes.to.exchange=Wishes To Exchange field should not be blank
```